RobotCode/robot.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
+import motion
+
+
+
+#middle_x = 430
+middle_xbasket = 424
```

Comment:
Compute based on camera spec, additionally this is calculated in multiple places